### PR TITLE
update readme

### PR DESCRIPTION
**Pull request by pritalpatel**
Pull Request for Issue #49.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -110,6 +110,8 @@ $ tests/vendor/bin/codecept run tests/acceptance/users.feature
 
 _If you want to see steps then you can use `--steps` option of codeceptio. Check [full codecept command list here](http://codeception.com/docs/reference/Commands#Run)_
 
+**Note**:You can modify the timeout time by changing the value of **TIMEOUT** constant lower for fast machines and higher for slow computers. The **TIMEOUT** constant located in file tests/acceptance/_bootstrap.php
```
Comment (yvesh) [inline README.md]: remove time (duplicate)

changing -> setting

is located in the following file:

File: README.md [hunk 2]
```diff
@@ -1,9 +1,9 @@
-[Browser Automated Tests for Joomla! CMS(covering Users and Content features - GSoC1 16)](https://summerofcode.withgoogle.com/projects/#5724182314745856)
+[Browser Automated Tests for Joomla! CMS(covering Users and Content features - GSoC 16)](https://summerofcode.withgoogle.com/projects/#5724182314745856)
 ===
 
 Abstract
 ---
-System development nowadays more than ever starts to look for automated test methods. There are several main drivers for this trend -need for faster design‐develop‐test‐analysis cycle, push for higher quality, increasing complexity of systems and their integration and last but not least ever‐rising costs of manual testing. Software testing itself is very wide field with topics ranging from compliance frameworks.
+System development nowadays more than ever starts to look for automated test methods. There are several main drivers for this trend need for faster design‐develop‐test‐analysis cycle, push for higher quality, increasing complexity of systems and their integration and last but not least ever‐rising costs of manual testing. Software testing itself is very wide field with topics ranging from compliance frameworks.
```
Comment (yvesh) [inline README.md]: The last sentence sounds like something is missing here (ranging from A, over b to C) :-)

As a general note try making that sentence a bit easier.

[https://www.youtube.com/watch?v=KC3hFsDozQs](https://www.youtube.com/watch?v=KC3hFsDozQs)

File: README.md [hunk 3]
```diff
@@ -86,7 +86,7 @@ _We are using `composer.json` file for `tests` folder, so that you will have to
 
 ### Run tests
 
-To run the tests please execute the following commands. We are usin [Robo.li](http://robo.li/) to execute the tests.
+To run the tests please execute the following commands. We are using [Robo.li](http://robo.li/) to execute the tests.
```
Comment (yvesh) [inline README.md]: Please also mention codeception somewhere :-) And that also the phpunit tests are run.

File: README.md [hunk 4]
```diff
@@ -86,7 +86,7 @@ _We are using `composer.json` file for `tests` folder, so that you will have to
...
 
 #### To execute all the test features you should use.
```
Comment (yvesh) [inline README.md:132]: System-tests, you forgot about the unit tests (and upcoming javascript tests)

File: README.md [hunk 5]
```diff
@@ -110,6 +110,8 @@ $ tests/vendor/bin/codecept run tests/acceptance/users.feature
 
 _If you want to see steps then you can use `--steps` option of codeceptio. Check [full codecept command list here](http://codeception.com/docs/reference/Commands#Run)_
 
+**Note**:You can modify the timeout time by setting the value of **TIMEOUT** constant lower for fast machines and higher for slow computers. The constant located in file tests/acceptance/_bootstrap.php
```
Comment (yvesh) [inline README.md]: constant is located in the file

File: README.md (hunk 2 above)
Comment (nibra) [inline README.md]: Make a bullet list of this:

> There are several main drivers for this trend
>      - need for faster design‐develop‐test‐analysis cycle,
>      - push for higher quality,
>      - increasing complexity of systems and their integration and last but not least
>      - ever‐rising costs of manual testing.

The last sentence is incomplete, like Yves already stated; also, I don't understand 'compliance frameworks' in this context.

Comment (pritalpatel) [inline README.md]: Ok, I will Remove the last sentence.

File: README.md (hunk 4 above)
Comment (pritalpatel) [inline README.md:132]: Humm but I thought this README.md is only focused for this project.

**Comment by javigomez** [comment]
Merging. Added todo's at https://github.com/joomla-projects/gsoc16_browser-automated-tests/issues/49 for future pulls

**Pull request merged**